api/v1/helper.go

```diff
@@ -801,3 +802,39 @@ func NetFilterMatch(netFilter string, netValue string) (isMatch bool) {
 
 	return netFilterResult[0][1] == netValueResult[0][1] && netFilterResult[0][2] == netValueResult[0][2]
 }
+
+// MaxUnavailable calculate the max number of unavailable nodes to represent the number of nodes
+// we can drain in parallel
+func (s *SriovNetworkPoolConfig) MaxUnavailable(numOfNodes int) (int, error) {
+	// this means we want to drain all the nodes in parallel
```

Comment:
Can that ever work? I think it would cause issues if it can't find any node to drain to. It may even lock up?

Reply:
sure this can work if you have a pool of worker nodes don't doesn't have any PDB pod running on them you can reboot all of them